tests/tests/test_state_scripts.py

```diff
@@ -632,15 +854,125 @@ class TestStateScripts(MenderTesting):
         "ArtifactFailure_Error_55", # Error for this state doesn't exist, should never run.
     ]
 
+    @pytest.mark.usefixtures("standard_setup_one_client_bootstrapped")
+    @pytest.mark.parametrize("description,test_set", REBOOT_TEST_SET)
```

Comment:
@kacf With the new reboot-detector, this turned out to be real simple, thus this is basically all there is to it,
the rest should look familiar to you, as I have merged this with ur master.